modules/mod_base/controllers/controller_postback.erl

```diff
@@ -28,6 +28,7 @@
 
 -include_lib("zotonic.hrl").
 
+-define(SIDEJOB_TIMEOUT, 1000).
```

Comment:
This is how long we wait for sidejobs handling the request. Any late data could be sent using comet, this wait period **must** be adjusted to be shorter than the wait period of the comet connection.